pvlib/pvsystem.py

```diff
@@ -2839,6 +2839,10 @@ def pvwatts_dc(g_poa_effective, temp_cell, pdc0, gamma_pdc, temp_ref=25.):
 
         P_{dc} = \frac{G_{poa eff}}{1000} P_{dc0} ( 1 + \gamma_{pdc} (T_{cell} - T_{ref}))
 
+    Note that the pdc0 is also used as a symbol in :py:func:`pvwatts_ac`. pdc0
+    in this function refers to the nameplate DC rating of the modules. pdc0 in
```

Comment:
`in this function refers to the DC power of the modules at reference conditions.